site/deploy-site.js

```diff
@@ -0,0 +1,52 @@
+const { execSync } = require('child_process');
+const process = require('process');
+const path = require('path');
+const args = require('minimist')(process.argv.slice(2));
+
+const execWithOutput = (cmd) => execSync(cmd, { stdio: 'inherit' });
+
+const DEPLOY_REPOSITORY = 'https://github.com/embark-framework/embark-site';
+const DEPLOY_REMOTE = 'embark-site';
+
+const TMP_DEPLOY_BRANCH = 'embark-site-deploy';
+const LOCAL_DEPLOY_BRANCH = 'embark-site-deploy-dist';
+const REMOTE_DEPLOY_BRANCH = 'gh-pages';
+const MASTER_BRANCH = 'master';
+
+const SITE_DIR = 'site';
+const PUBLIC_DIR = 'public';
+const ROOT_DIR = path.join(__dirname, '..');
+
+const remote = args['deploy-remote'] || DEPLOY_REMOTE;
+const repository = args['deploy-repository'] || DEPLOY_REPOSITORY;
+const remoteDeployBranch = args['deploy-branch'] || REMOTE_DEPLOY_BRANCH;
+
+function main() {
+  const hasCorrectOrigin = execSync(`git remote -v | grep ${remote}`).toString().indexOf(remote) > -1;
+  const hasCorrectOriginRepo = execSync(`git remote -v | grep ${remote}`).toString().indexOf(repository) > -1;
```

Comment:
```suggestion
  const hasCorrectRemoteRepo = remote !== DEPLOY_REMOTE ? true : execSync(`git remote -v | grep ${remote}`).toString().indexOf(repository) > -1;
```

See: https://github.com/embark-framework/embark/pull/1522/files#r278630603.